### ISSUE TITLE
Sixth week maintenance - week 38 2024

### ISSUE THREAD

**Issue by henkvancann** [WebOfTrust/WOT-terms#188]
To do:

- Scraper - maintenance
- Document the data level in the article tag, how it got there, how persistent it is, and its use and benefits. (https://github.com/WebOfTrust/WOT-terms/issues/139)
Update this list of https://github.com/WebOfTrust/WOT-terms/issues/138 according to current affairs
- Review markdown files on https://blockchainbird.github.io/spec-up-t-website/ via Grammarly

**Comment by henkvancann** [comment]
Excluding these markdown files, because already done: https://github.com/blockchainbird/spec-up-t-website/issues/6#issuecomment-2298846766

**Comment by kordwarshuis** [comment]
Read this before starting: https://github.com/blockchainbird/spec-up-t-website/issues/6#issuecomment-2298810078
